include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3219,6 +3219,8 @@ struct OrtApi {
   * \param[in] cuda_options
   *
   * \snippet{doc} snippets.dox OrtStatus Return Value
+  *
+  * \version New in version 1.11.
```

Comment:
I don't think we want to say 'New in version 1.11' since that might sound weird after we're on 1.12.

Maybe even just `\version 1.11` would be enough? I tried it locally and it looked clear and concise.

Reply:
Another idea is to adopt a word used in other code documentation systems is "since" e.g. javadoc (https://www.baeldung.com/javadoc-version-since)

It would become `\since version 1.11`

Reply:
What should we do when we remove APIs in the future? Looks like `\since` isn't setup to have an ending.

Reply:
I suppose that text would have to go elsewhere, as the API being documented wouldn't be there anymore right?

Reply:
Do we ever remove APIs? ie how far back does our backwards compatibility commitment go?